bf2fc6cc711aee1a0c2a/cos-observability-resources
# Completely remove prometheus rules from dataplanes

## Description

Previous work enabled prometheus rules in Observatorium. With rules running in Observatorium, there is no need to have them running in the dataplanes.

This PR removes the rules files and deletes references to those files
## Verification Steps

* Go to a dev cluster, change the [cos-fleetshard-observability-upstream](https://console-openshift-console.rhoc-dev-153f1de160110098c1928a6c05e19444-0000.eu-de.containers.appdomain.cloud/k8s/ns/redhat-openshift-connectors-observability/secrets/cos-fleetshard-observability-upstream) secret set the repository and branch to the one containing this changes.
* Delete the Observability resource
* Restart the sync pod, change the Observability resource setting `disablePagerDuty: true`. [This change will be set by the syn in the next release of the addon](https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/pull/583/files#diff-5f2a8a19ccc64aa9685b595876c0f6590f26dfd0a018df085dafef25b840d857L118)
* Check that prometheus is up and running, and that it contains no rules

## Checklist (Definition of Done)


~~- [ ] Required Standard Operating Procedure (SOP) is added.~~

## Prometheus

- [x] Changes were tested against dev environment

## Grafana

~~- [ ] You have imported the changes into a dev or staging environment~~
